index.html

```diff
@@ -1189,7 +1190,9 @@
       interface NDEFRecord {
         readonly attribute NDEFRecordType recordType;
         readonly attribute USVString mediaType;
-        NDEFRecordData data();
+        [NewObject] USVString toText();
```

Comment:
You might want to check this, but NewObject might not apply to `USVString`... I'm not sure  tho.

Reply:
(Strings are primitives, so they are always passed by value... which is why NewObject seems suspect/redundant here)

Reply:
OK, I will remove the `[NewObject]`.

Reply:
Done.